src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -23,7 +23,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj" />
-    <ProjectReference Include="..\MSBuild.BuildHost\Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj" />
```

Comment:
Unfortunately NuGet doesn't give another way to opt out of this -- there's a TODO tracking what I actually want:

https://github.com/NuGet/NuGet.Client/blob/29f292950dfa50aea030bd74c3277e983eff711e/src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTaskLogic.cs#L888-L890